### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by GriffinWhiting**

### CODE REVIEW

File: projects/bagel-finance/index.js [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+             {"0":"0x63cA00DEfc958af2a8a14923978F79CAd1268fE4","1":"0x340192d37d95fb609874b1db6145ed26d1e47744"},
+             {"0":"0xA37255b10b1757aA1c6009A3089F50b654f5a093","1":"0xaf9aa53146c5752bf6068a84b970e9fbb22a87bc"},
+             {"0":"0xBd245975DB5D4255E5E6c4531DffE5A6829db985","1":"0xba68d6bee4f433630dee22c248a236c8f6eae246"},
+             {"0":"0xe4aD6C2Cd37DEd0560f766327dD426e04B0Fa657","1":"0x1c0276642f2a7cbcf6624d511f34811cdc65212c"},
+             {"0":"0x3b402F7448edC7c7a959B336367a06aDDe9Ca07B","1":"0x223740a259e461abee12d84a9fff5da69ff071dd"},
+             {"0":"0xdf7039684F943E82a8738A340830e4D8068F1f7D","1":"0x969f2556F786a576F32AeF6c1D6618f0221Ec70e"},
+             {"0":"0x514b825ebb062EB29681C281092f27eFeC838Ceb","1":"0x82E8F9e7624fA038DfF4a39960F5197A43fa76aa"},]
+async function bnbTvl(bankAddress, block) {
+  let balances = {
+    "0x0000000000000000000000000000000000000000": (
```
Comment (0xngmi) [inline projects/bagel-finance/index.js:35]: this should be bnb's address

File: projects/bagel-finance/index.js [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+    })),
+    chain: 'bsc'
+  })).output
+
+  const lendingBalances = banksInfos.filter((n) => {
+    return true
+  }).map((n) => {
+    const stakingTokenAddr = n.input.params[0];
+    const i5 = n.output['5'];
+    const i6 = n.output['6'];
```
Comment (0xngmi) [inline projects/bagel-finance/index.js:65]: everything looks good but including borrowed funds here concerns me, afaik borrowed funds are used in the lps that we count later in this function, so counting them here as well might easily lead to double counting

File: projects/bagel-finance/index.js [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+  const lendingBalances = banksInfos.filter((n) => {
+    return true
+  }).map((n) => {
+    const stakingTokenAddr = n.input.params[0];
+    const i5 = n.output['5'];
+    const i6 = n.output['6'];  
+    const i8 = n.output['8'];
+    return {
+      token: stakingTokenAddr,
+      balance: BigNumber(i5).plus(BigNumber(i6)).plus(BigNumber(i6)).plus(BigNumber(i8)).plus(BigNumber(i6)).toFixed(0)
```
Comment (0xngmi) [inline projects/bagel-finance/index.js:69]: why is i6 added 3x?

**Comment by GriffinWhiting** [comment]
Because our average borrowing rate is three times, this place needs x3 liang chen ***@***.*** 签名由 网易邮箱大师 定制 On 11/05/2021 03:14, 0xngmi wrote: @0xngmi commented on this pull request. In projects/bagel-finance/index.js: > +      target: bankAddress,
+      params: token
+    })),
+    chain: 'bsc'
+  })).output
+
+  const lendingBalances = banksInfos.filter((n) => {
+    return true
+  }).map((n) => {
+    const stakingTokenAddr = n.input.params[0];
+    const i5 = n.output['5'];
+    const i6 = n.output['6'];  
+    const i8 = n.output['8'];
+    return {
+      token: stakingTokenAddr,
+      balance: BigNumber(i5).plus(BigNumber(i6)).plus(BigNumber(i6)).plus(BigNumber(i8)).plus(BigNumber(i6)).toFixed(0)
 why is i6 added 3x? — You are receiving this because you authored the thread. Reply to this email directly, view it on GitHub, or unsubscribe. Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by 0xngmi** [comment]
understood, could you comment on borrowed coins being double counted as part of LPs?

**Pull request merged**